lib/portage/dbapi/bintree.py

```diff
@@ -1234,11 +1234,33 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", "true")
+        # getuto is a shell script...
+        ret = os.system(portage_trust_helper)
+        if ret == 32512:
```

Comment:
Could you try https://docs.python.org/3/library/os.html#os.waitstatus_to_exitcode?

Reply:
Done